common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -176,11 +176,12 @@ def do_my_expensive_update(course_key):
     course_deleted = SwitchedSignal("course_deleted", providing_args=["course_key"])
     library_updated = SwitchedSignal("library_updated", providing_args=["library_key"])
     item_deleted = SwitchedSignal("item_deleted", providing_args=["usage_key", "user_id"])
+    grading_policy_change = SwitchedSignal("grading_policy_change", providing_args = ["course_key"])
```

Comment:
We can come back and change this later, right? We'll probably at least want to think about adding more args for eventing, like the user who initiated the change and stuff.